apps/web/src/components/Displays/BlobGasUsageDisplay.tsx

```diff
@@ -1,40 +1,63 @@
 import type { FC } from "react";
 import cn from "classnames";
 
-import {
-  BLOB_GAS_LIMIT_PER_BLOCK,
-  TARGET_BLOB_GAS_PER_BLOCK,
-  calculateBlobGasTarget,
-  calculatePercentage,
-  formatNumber,
-} from "~/utils";
+import type { NetworkBlobConfig } from "@blobscan/network-blob-config";
+
+import { calculatePercentage, formatNumber, performDiv } from "~/utils";
 import { PercentageBar } from "../PercentageBar";
 
 type BlobGasUsageDisplayProps = {
+  networkBlobConfig: NetworkBlobConfig;
   blobGasUsed: bigint;
   compact?: boolean;
 };
 
-function getTargetSign(blobGasUsed: bigint): "+" | "-" | undefined {
-  if (blobGasUsed < TARGET_BLOB_GAS_PER_BLOCK) {
+function getTargetSign(
+  blobGasUsed: bigint,
+  targetBlobGasPerBlock: bigint
+): "+" | "-" | undefined {
+  if (blobGasUsed < targetBlobGasPerBlock) {
     return "-";
-  } else if (blobGasUsed > TARGET_BLOB_GAS_PER_BLOCK) {
+  }
+
+  if (blobGasUsed > targetBlobGasPerBlock) {
```

Comment:
Does it make sense to have undefined as possible return value? This comparison could just be >=